lib/stylus-mixins/units/em.styl

```diff
@@ -1,11 +1,23 @@
 //
 // EM
-// ==
+// ===
```

Comment:
Can you revert this line please? Our 'underlines' normally match the title length.